magefiles/mage.go

```diff
@@ -18,14 +18,14 @@ import (
 
 const (
 	registryPrefix         = "ghcr.io/nordsecurity/nordvpn-linux/"
-	imageBuilder           = registryPrefix + "builder:1.1.5"
-	imagePackager          = registryPrefix + "packager:1.0.3"
+	imageBuilder           = registryPrefix + "builder:1.2.0"
+	imagePackager          = registryPrefix + "packager:1.1.0"
 	imageSnapPackager      = registryPrefix + "snaper:0.0.2"
```

Comment:
snaper/Docker file is modified - probably should be new version?

Reply:
It will not result in changes in image as no changes were made for docker comands